### PR TITLE
Transaction: Update design on table head

### PR DESCRIPTION
**Pull request by kosssi**
Changes :
- Add a flag (transaction-history) to display separately the graph current balance and transaction
- Add blue frame on transaction
- Add new design on table head

You can see design with and without `transaction-history` flag:

without | with
--------|-----
capture d ecran 2018-12-06 a 09 00 24 | capture d ecran 2018-12-06 a 09 01 19
capture d ecran 2018-12-06 a 09 02 39 | capture d ecran 2018-12-06 a 09 01 47

### CODE REVIEW

File: src/components/Header/Header.styl [hunk 1]
```diff
@@ -1,5 +1,15 @@
 @require 'settings/breakpoints'
 
+.HeaderFixed
+    position sticky
```
Comment (drazik) [inline src/components/Header/Header.styl]: Beware, it doesn't work in IE11. Do we have officially dropped the support of IE? I remember there's a Paper about it but I can't find it.

File: src/ducks/transactions/TransactionsPage.jsx [hunk 2]
```diff
@@ -243,24 +242,20 @@ class TransactionsPage extends Component {
     }
 
     return (
-      <div
-        className={styles.TransactionPage__bottom + ' js-transactionPageBottom'}
```
Comment (drazik) [inline src/ducks/transactions/TransactionsPage.jsx]: This element was used in https://github.com/cozy/cozy-banks/blob/master/src/ducks/transactions/Transactions.jsx#L136-L143, but I don't see the corresponding change in the PR. Is it normal? Does the infinite scroll still work?

Comment (kosssi) [reply src/ducks/transactions/TransactionsPage.jsx]: I change it with `js-scrolling-element`

File: src/components/Header/Header.styl (hunk 1 above)
Comment (kosssi) [reply src/components/Header/Header.styl]: I think not

Comment (drazik) [reply src/components/Header/Header.styl]: Talked about it on mattermost. We don't support IE anymore so it's OK :)

File: src/ducks/transactions/TransactionsPage.jsx (hunk 2 above)
Comment (drazik) [reply src/ducks/transactions/TransactionsPage.jsx]: Oh yes sorry I missed the change in which you update the `querySelector`.

**Pull request merged**